packages/google_maps_flutter/lib/src/webview.dart

```diff
@@ -0,0 +1,840 @@
+// Copyright 2021 Samsung Electronics Co., Ltd. All rights reserved.
+// Copyright 2013 The Flutter Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+part of google_maps_flutter_tizen;
```

Comment:
Why should this plugin have their own Webview implementation?

Reply:
There is no special reason. However, this plugin uses platform view components (like AndroidView and UiKitView based on PlatformView) to present a google_map. However, because there is no TizenView on flutter engine, I added it in this plugin.

Reply:
Can't we use webview_flutter tizen implementation?

Reply:
To use webview plugin, first of all, it needs to be updated to latest version. Applying to the latest version for webview plugin may be done later.

Reply:
Can we remove webView implementation from this plugin after migrating webview_flutter to the latest version?
Thank you for your answer.

Reply:
Though I am not sure because I have not tried it, it doesn't seems impossible.